frontend/src/components/core/Sidebar/SidebarNav.test.tsx

```diff
@@ -17,12 +17,10 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 
 import { matchers } from "@emotion/jest"
-import { ExpandMore, ExpandLess } from "@emotion-icons/material-outlined"
 import React from "react"
 import * as reactDeviceDetect from "react-device-detect"
 import { act } from "react-dom/test-utils"
 
-import Icon from "src/components/shared/Icon"
 import { useIsOverflowing } from "src/lib/Hooks"
 import { mount, shallow } from "src/lib/test_util"
```

Comment:
Quick note: it is our goal on the Open Source team to transition away from using Enzyme so that we can upgrade to React v18, which does not support the Enzyme testing framework. This does not impact usage of Jest.

We have started asking the following for PR approval:
* All new FE unit tests be written using `react-testing-library` (RTL) instead -> removing usage of the `mount` and `shallow` functions, `.find`, etc. 
* If you are editing unit tests, rewrite the enzyme tests to RTL

I realize this might be an aggressive ask in the case of this PR and depending on your familiarity with RTL/Enzyme, so happy to make the exception, but want to take the opportunity to share some resources for future reference:
* [RTL Docs Migration Example](https://testing-library.com/docs/react-testing-library/migrate-from-enzyme/#basic-enzyme-to-react-testing-library-migration-examples)
* Some conversions shown in [this PR](https://github.com/streamlit/streamlit/pull/6084/files)

Reply:
Got it, thanks for letting me know, and thanks for suggesting the exception ❤️ 

We can talk a bit more tomorrow during our meeting and decide whether we might do it for this PR or not, if that works for you